internal/pkg/state/load_remote_test.go

```diff
@@ -461,5 +461,5 @@ func loadRemoteState(t *testing.T, m *manifest.Manifest, projectStateFile string
 	filter := m.Filter()
 	_, localErr, remoteErr := state.Load(LoadOptions{RemoteFilter: filter, LoadRemoteState: true})
 	assert.NoError(t, localErr)
-	return state, envs, remoteErr
+	return state, testProject.Env(), remoteErr
```

Comment:
Do `GetTestProject(t, envs)` vstupuju premenne prostredia ... doteraz to bolo tak, ze sa k tymto env pridavali dalsie ohladom testovacieho projektu ... nebolo to velmi intuitivne, ... ze tam vlozis `envs` a oni sa zmenia.

Teraz je to uz immutable.